packages/dds/tree/src/core/forest/forest.ts

```diff
@@ -34,6 +34,11 @@ import type { IEditableForest } from "./editableForest.js";
  * @internal
  */
 export interface ForestEvents {
+	/**
+	 * A new root field was just created in this forest.
+	 */
+	afterCreateRootField(key: FieldKey): void;
```

Comment:
Nit: I'd name it `rootFieldCreated` but no strong feelings about it, especially since it is internal.
```suggestion
	afterCreateRootField(key: FieldKey): void;
```

Reply:
I'm not sure if we'll ever need a "before" variant, but I do think that the "after" makes it easy to know what's happening at a glance (is the root already in the forest? Or not yet?). In either case, I think it's okay for a low-level, internal event. If it were a customer-facing event, I'd probably try to have a "nicer" name, even at the cost of immediate clarity. 

I will rename it to "afterRootFieldCreated" though, to use your suggestion, because that sounds nicer.